experimental/parser/testdata/parser/method/incomplete.proto.stderr.txt

```diff
@@ -8,13 +8,13 @@ error: missing method return type in service method
   --> testdata/parser/method/incomplete.proto:21:13
    |
 21 |     rpc Bar2(foo.Bar);
-   |             ^^^^^^^^^ expected type in `(...)` after this
+   |             ^^^^^^^^^ expected return type in `returns (...)` after this
```

Comment:
I don't really understand what this is trying to say.

This might be simpler to just omit the "return type in", too:
```
expected `returns (...)` after this
```